src/subcommands/check.rs

```diff
@@ -73,47 +86,45 @@ pub fn command(display_order: usize) -> Command<'static> {
         )
 }
 
-pub fn run(matches: &ArgMatches) -> bool {
+pub fn run(matches: &ArgMatches) -> Result<(), Error> {
     let path = matches.value_of(DB_PATH).unwrap();
     let failfast = !matches.is_present(NO_FAILFAST);
     let specific = matches.value_of(SPECIFIC);
     let start_at: usize = matches
         .value_of(START_AT)
-        .unwrap()
+        .expect("should have a default")
         .parse()
         .expect("Value of \"--start-at\" must be an integer.");
```

Comment:
```suggestion
        .unwrap_or_else(|_| panic!("Value of \"--{}\" must be an integer.", START_AT));
```

A nit, just to avoid hardcoding.

Reply:
Done.